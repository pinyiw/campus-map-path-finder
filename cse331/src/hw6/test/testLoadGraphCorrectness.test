# load graph
LoadGraph g heroes.tsv

# check graph correctness
# list nodes
ListNodes g

# list children
ListChildren g Java
ListChildren g C
ListChildren g Logic
ListChildren g Data
ListChildren g Math
ListChildren g Calculus