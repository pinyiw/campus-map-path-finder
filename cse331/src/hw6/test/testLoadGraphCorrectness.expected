# load graph
loaded graph g

# check graph correctness
# list nodes
g contains: C Calculus Data Java Logic Math

# list children
the children of Java in g are: Data(CSE143) Data(CSE332) Logic(CSE142) Logic(CSE143) Logic(CSE331) Logic(CSE332)
the children of C in g are:
the children of Logic in g are: Data(CSE143) Data(CSE332) Java(CSE142) Java(CSE143) Java(CSE331) Java(CSE332) Math(CSE312)
the children of Data in g are: Java(CSE143) Java(CSE332) Logic(CSE143) Logic(CSE332)
the children of Math in g are: Calculus(Math126) Logic(CSE312)
the children of Calculus in g are: Math(Math126)